src/Lykke.Logs/LykkeLogToAzureSlackNotificationsManager.cs

```diff
@@ -137,5 +151,26 @@ private HashSet<string> DefaultLogLevelsInit()
                 LykkeLogToAzureStorage.MonitorType,
             };
         }
+
+        private bool IsSameMessage(LogLevel level, string message)
+        {
+            var now = DateTime.UtcNow;
+            if (_lastTimes.TryGetValue(level, out DateTime lastTime))
+            {
+                if (_lastMessages.TryGetValue(level, out string lastMessage))
+                {
+                    if (lastMessage == message && now - lastTime < _sameMessageMutePeriod)
+                        return true;
+                    _lastMessages.TryUpdate(level, message, lastMessage);
```

Comment:
No need to update. The message is the same

Reply:
Not agree, message could be different here